.github/workflows/qe-image.yml

```diff
@@ -0,0 +1,55 @@
+name: Push qe images for new tag
+on:
+  push:
+    tags:
+      - '*'
+env:
+  REGISTRY_USER: "crcont+qeghcrc"
```

Comment:
Add as secret as well (We can re-create the robot account)

Reply:
updated